### PR TITLE
Issue defined jsonschema

### PR DESCRIPTION
**Pull request by BrandonRudiselNDSU**
Working on #12 and #1

### CODE REVIEW

File: examples/events/EiffelIssueDefinedEvent/simple.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+	"meta": {
+		"type": "EiffelIssueDefinedEvent",
+		"version": "1.0.0",
+		"time": 1234567890,
+		"id": "aaaaaaaa-bbbb-5ccc-8ddd-eeeeeeeeeee0"
+	},
+	"data": {
+		"type": "BUG",
+		"tracker": "DONE", //the hell's a tracker?
```
Comment (jaden-young) [inline examples/events/EiffelIssueDefinedEvent/simple.json]: `"tracker"` should be the name of an issue tracker, e.g. JIRA, Gerrit, Bugzilla, etc.

Comment (jaden-young) [reply examples/events/EiffelIssueDefinedEvent/simple.json]: Just make it "github" here and we'll make the URI look the same

File: examples/events/EiffelIssueDefinedEvent/simple.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+		"id": "aaaaaaaa-bbbb-5ccc-8ddd-eeeeeeeeeee0",
+		"uri": "http://zombo.com/"
```
Comment (jaden-young) [inline examples/events/EiffelIssueDefinedEvent/simple.json]: `"uri": "https://github.com/my-user/my-repo/issues/3"`

File: examples/events/EiffelIssueDefinedEvent/simple.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+	},
+	"links": [
+		{
+			"type": "BUG",
```
Comment (jaden-young) [inline examples/events/EiffelIssueDefinedEvent/simple.json]: `"BUG"` isn't a valid link type, we can get rid of it here.

**Review by jaden-young — CHANGES_REQUESTED** [review]

**Comment by jaden-young** [comment]
Note that work for issue #12 has been folded into this PR

**Comment by jaden-young** [comment]
Since we're also taking care of #12 here, we're going to want to take care of its other task: adding a link to the example file in the EiffelIssueDefinedEvent markdown document.

File: examples/events/EiffelIssueDefinedEvent/simple.json (hunk 2 above)
Comment (BrandonRudiselNDSU) [reply examples/events/EiffelIssueDefinedEvent/simple.json]: But at zombo com you can do anything.

File: examples/events/EiffelIssueDefinedEvent/simple.json (hunk 3 above)
Comment (BrandonRudiselNDSU) [reply examples/events/EiffelIssueDefinedEvent/simple.json]: By get rid of, do you mean remove the line, or remove the block?

Comment (jaden-young) [reply examples/events/EiffelIssueDefinedEvent/simple.json]: The block.

File: examples/events/EiffelIssueDefinedEvent/simple.json (hunk 2 above)
Comment (jaden-young) [reply examples/events/EiffelIssueDefinedEvent/simple.json]: I'm afraid I don't understand the reference, but we should use something illustrative.

**Review by jaden-young — APPROVED** [review]

**Review by jaden-young — CHANGES_REQUESTED** [review]
Oops, almost forgot. We need to add a link to the example from the `eiffel-vocabulary/EiffelIssueDefinedEvent.md` file

**Review by jaden-young — APPROVED** [review]

**Pull request merged**